pkg/nsx/services/inventory/pod.go

```diff
@@ -133,3 +136,36 @@ func (s *InventoryService) CleanStaleInventoryApplicationInstance() error {
 	}
 	return nil
 }
+
+func (s *InventoryService) CleanUpInventoryApplicationInstance(ctx context.Context) error {
+	log.Info("Cleanup InventoryApplicationInstance")
+	containerApplicationInstances := s.ApplicationInstanceStore.List()
```

Comment:
add return if len is 0

Reply:
if len == 0, nothing will be done. but return  will be more clear